corehq/apps/hqwebapp/management/commands/build_requirejs.py

```diff
@@ -228,6 +227,36 @@ def _minify(self, config):
             if ret:
                 raise CommandError(f"Failed to minify {rel_path}")
 
+    def _copy_modules_back_into_corehq(self, config, local_js_dirs):
+        """
+        Copy optimized modules in staticfiles back into corehq
+        """
+        if not self.local:
```

Comment:
This would prevent your local git pollution you mentioned in the meeting, right?

Reply:
yes, the git pollution only happens when `local` is set